platform/nodejs/main.js

```diff
@@ -146,6 +146,14 @@ function reset() {
     snfe.reset();
 }
 
+// rollup.js needs module.exports to be set back to the local exports object.
+// This is because some of the code (e.g. publicsuffixlist.js) sets
+// module.exports. Once all included files are written like ES modules, using
+// export statements, this should no longer be necessary.
+if (typeof module !== 'undefined' && typeof exports !== 'undefined') {
+  module.exports = exports;
```

Comment:
@gorhill sorry I used the wrong indentation level here.

I notice a few [jshint](https://jshint.com/) comments in the source. Are you using this tool? Maybe you should add the configuration to the repo along with a `lint` target in `Makefile`.

Reply:
[`.jshintrc` is already present](https://github.com/gorhill/uBlock/blob/master/.jshintrc) in the repo.

As for the `lint` command, I am going to have to read to find out best way to do this -- [here](https://coderwall.com/p/zqcghq/use-jshint-with-node-js) is says to add

    lint:
        ./node_modules/.bin/jshint ...

Which means I need to add a `jshint` target as well I guess? My installation of `jshint` is outside the ublock project and I use a specific path to access the command in my text editor, not suitable for the public Makefile here.

Reply:
> [`.jshintrc` is already present](https://github.com/gorhill/uBlock/blob/master/.jshintrc) in the repo.

Sorry, my bad!

> Which means I need to add a `./node_modules/.bin/jshint` target as well I guess? My installation of `jshint` is outside the ublock project and I use a specific path to access the command in my text editor, not suitable for the public Makefile here.

It's OK to expect people to install jshint manually via `npm install -g jshint`. Maybe this should be added to the `CONTRIBUTING.md`.

As for the `lint` target, let me see how this works on the command line since I haven't used it.

Reply:
It seems JSHint removed indentation level checks a long time ago: https://github.com/jshint/jshint/releases/tag/2.5.0

I'll see if I can add some basic [ESLint](https://eslint.org/) configuration that matches the JSHint configuration and also adds options for indentation, etc. to match the current coding style.